.github/workflows/ci.yaml

```diff
@@ -22,15 +22,18 @@ jobs:
   build:
     needs: ci
     runs-on: ubuntu-latest
+    env:
+      TRAVIS_BRANCH: ${{ github.head_ref }}
```

Comment:
On a PR this will be the branch name, on a non-PR, this will be blank, but then set later in the Set ENV variables section.

Reply:
What do you think of removing this and just using the `Set ENV variables` step?  Then it be `master` or `240`, right?

Reply:
Let me see if I can play with that - I'm playing around in https://github.com/Fryguy/sandbox

Reply:
Done